packages/ocephes/ocephes.0.0.1/descr

```diff
@@ -0,0 +1,4 @@
+Bindings to special math functions from the Cephes library.
```

Comment:
@Chris00  That's a good point regarding the license. It seems like lots of other libraries (specifically of the Python & Lua variety) end up repackaging it in their source code. I'm going to follow their example and email the author for permission. In the meantime, I'm going to close this pull request.